cpu/lpc1768/Makefile.features

```diff
@@ -1,3 +1,5 @@
+# CPU_MODEL is the same as the CPU for this board
```

Comment:
~~board~~ architecture

Reply:
Indeed `board` is wrong here.
I am not sure about `family` though.

The fact that it is the same is only a `RIOT` directory naming thing.
`CPU` only means the directory inside `cpu` and has no meaning outside.

So maybe the other way around, `The name of the CPU is the CPU_MODEL` or something ?

Reply:
Choose whatever you like best, I just wanted to say board is wrong here.

Reply:
> `The name of the CPU is the CPU_MODEL`

tbh my head hurts a bit when reading that.

How about

>  `This CPU only implements one CPU_MODEL`

Reply:
I thought about this one, but the CPU could still be named differently.

`This CPU only implements one CPU_MODEL with the same name` ?

Reply:
Sounds good, no need for a separate !fixup commit - this PR is small enough.